### PR TITLE
[Feature2] Second feature for this Project

### PR DESCRIPTION
**Pull request by rajpatil09**
1. added feature-2 folder.
2. added text file into the folder.

### CODE REVIEW

File: feature-two/feature2-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was added in the feature2 branch.
```
Comment (rajpatil87) [inline feature-two/feature2-file.txt:1]: It was a good idea to create another feature.

File: feature-two/feature2-file.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (rajpatil87) [inline feature-two/feature2-file.txt:2]: This line is kept empty for no reason.

File: feature-two/feature2-file.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+Another change, added one more line.
```
Comment (rajpatil87) [inline feature-two/feature2-file.txt:3]: three lines are enough in this file.

**Review by rajpatil87 — APPROVED** [review]

File: feature-two/feature2-file.txt (hunk 1 above)
Comment (rajpatil09) [reply feature-two/feature2-file.txt:1]: I agree

File: feature-two/feature2-file.txt (hunk 2 above)
Comment (rajpatil09) [reply feature-two/feature2-file.txt:2]: Accidentally hit the enter key

File: feature-two/feature2-file.txt (hunk 3 above)
Comment (rajpatil09) [reply feature-two/feature2-file.txt:3]: Yup

**Comment by rajpatil09** [comment]
Done with the merging

**Comment by rajpatil87** [comment]
Great

**Pull request merged**